courses/intro-to-ada/index.rst

```diff
@@ -0,0 +1,20 @@
+.. intro-to-ada documentation master file, created by
+   sphinx-quickstart on Mon Feb 19 11:39:35 2018.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+Introduction to Ada
+===================
+
+.. toctree::
+    :maxdepth: 2
+    :caption: Contents:
+
+    intro_to_ada
+
+Indices and tables
```

Comment:
Do you need an index here, since both course are part of an enclosing one?
Are these local index only reporting on local stuff?